README.md

```diff
@@ -41,6 +61,31 @@ rake yarn:install
 rails db:setup
 ```
 
+##### Issues with MacOS Catalina and Puma
+
+If you encounter issues installing the `puma` gem on MacOS Catalina, it may be due to compiler upgrades Apple issued.
+
+You may see errors like this:
```

Comment:
We could just note which step in the setup process raises these errors? I'm assuming `bin/setup`?